styleguide/derek/_partials/solutions/compare-table.ejs

```diff
@@ -0,0 +1,176 @@
+<table class="compareTable">
+
+  <thead class="compareTable-thead">
+    <tr class="compareTable-theadRow">
+      <td class="compareTable-theadCell compareTable-theadCell-long">Human Experts</td>
+      <td class="compareTable-theadCell">Navigator</td>
+      <td class="compareTable-theadCell">Aviator</td>
+    </tr>
+  </thead>
+
+  <tbody class="compareTable-body">
+
+    <tr class="compareTable-bodyRow">
+      <td class="compareTable-bodyCell">
+        <strong>Technical Account Manager (TAM)</strong>
+        <ul class="compareTable-list">
+          <li class="compareTable-listItem">Personal contact for ongoing business and technical assistance</li>
+        </ul>
+      </td>
+      <td class="compareTable-bodyCell">
+        <i class="fa fa-check check-green"></i>
+      </td>
+      <td class="compareTable-bodyCell">
+        <i class="fa fa-check check-green"></i>
+      </td>
+    </tr>
+
+    <tr class="compareTable-bodyRow">
+      <td class="compareTable-bodyCell">
+        <strong>Support of the AWS platform</strong>
+        <ul class="compareTable-list">
+          <li class="compareTable-listItem">100% certified AWS engineers deliver Fanatical Support for AWS 24x7x365</li>
+        </ul>
+      </td>
+      <td class="compareTable-bodyCell">
+        <i class="fa fa-check check-green"></i>
```

Comment:
That should have been removed all together. It was a carry over from the old compare tables. They are now removed